deps/rabbitmq_management/src/rabbit_mgmt_login.erl

```diff
@@ -38,3 +39,6 @@ login(<<"POST">>, Req0, State) ->
 login(_, Req0, _) ->
   %% Method not allowed.
 	cowboy_req:reply(405, Req0).
+
+get_authorization_token(undefined, Access_Token_Form_Field) -> Access_Token_Form_Field;
```

Comment:
Erlang uses CamelCase for bindings (variables), so this should be `AccessTokenFormField`.

Reply:
Thanks @michaelklishin !